src/test/java/ogc/rs/restAssuredTest/TilesMetaDataOnboardingProcessIT.java

```diff
@@ -0,0 +1,265 @@
+package ogc.rs.restAssuredTest;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.vertx.core.json.JsonObject;
+import jdk.jfr.Description;
+import ogc.rs.util.FakeTokenBuilder;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.UUID;
+
+import static io.restassured.RestAssured.given;
+import static ogc.rs.common.Constants.*;
+import static ogc.rs.processes.tilesMetaDataOnboarding.Constants.*;
+import static ogc.rs.processes.collectionOnboarding.Constants.RESOURCE_OWNERSHIP_ERROR;
+import static ogc.rs.processes.util.Status.ACCEPTED;
+import static ogc.rs.restAssuredTest.Constant.*;
+import static org.hamcrest.Matchers.is;
+
+@ExtendWith(RestAssuredConfigExtension.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class TilesMetaDataOnboardingProcessIT {
+    private static final Logger LOGGER = LogManager.getLogger(CollectionAppendingProcessIT.class);
+    String executionEndpoint = "/processes/{processId}/execution";
+    String jobStatusEndpoint = "/jobs/{jobId}";
+    String processId = "a512b49a-f8e6-4da1-99b0-13aa758c9104";
+
+
+    @BeforeAll
+    public static void setup() throws IOException {
+        // Set up files for the tests
+        // put the file in /fileName path
+        File validFile = new File("src/test/resources/processFiles/c432b3df-0a22-485d-80a5-ace8195f6074/WebMercatorQuad/0/0/0");
+        given().port(PORT).body(validFile).when().put(BUCKET_PATH + "src/test/resources/processFiles/c432b3df-0a22-485d-80a5-ace8195f6074/WebMercatorQuad/0/0/0")
```

Comment:
Do not include `src/test/resources/processFiles` here - try to put it from `c4...` onwards, same as line 43-44. Make this change and see if it works.